core-bundle/src/Resources/contao/library/Contao/Database/Statement.php

```diff
@@ -225,7 +231,7 @@ public function limit($intRows, $intOffset=0)
 	 */
 	public function execute()
 	{
-		return $this->query('', array_merge($this->arrSetParams, \func_get_args()));
+		return $this->query('', \func_get_args());
```

Comment:
This cannot be removed here, as it would break how `query()` works currently.

Instead the types have to be passed here too.

```suggestion
 		return $this->query('', array_merge($this->arrSetParams, \func_get_args()), $this->arrSetTypes);
```